src/indra_cogex/apps/queries_web/__init__.py

```diff
@@ -75,11 +248,11 @@ def get_example_data():
     "agent": fields.String(example="MEK"),
     "other_agent": fields.String(example="ERK"),
     "agent_role": fields.String(example="Subject"),
-    "other_role" : fields.String(example="Object"),
+    "other_role": fields.String(example="Object"),
     "stmt_source": fields.String(example="reach"),
     "stmt_sources": fields.List(fields.String, example=["reach", "sparser"]),
     "include_db_evidence": fields.Boolean(example=True),
-    "cell_line": fields.List(fields.String, example=["CCLE", "BT20_BREAST"]),
+    "cell_line": fields.List(fields.String, example=["CCLE", "BT20_BREAST", "HEL_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE"]),
```

Comment:
If you want to change the example you need to replace the identifier, not add to the list.
```suggestion
    "cell_line": fields.List(fields.String, example=["CCLE", "HEL_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE"]),
```
So keep either of `"BT20_BREAST"` or `"HEL_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE"`, but not both.

Reply:
Fixed in 4640b451